ci-operator/config/open-cluster-management/website/OWNERS

```diff
@@ -0,0 +1,5 @@
+approvers:
+- qiujian16
+
+reviewers:
+- qiujian16
```

Comment:
Please add more people. Who will review your PRs?

Reply:
added

Reply:
Not to this OWNERS file though :)

Reply:
ah sorry, added to this file also.